yrs/src/update.rs

```diff
@@ -181,7 +181,7 @@ impl Update {
             let mut store = txn.store_mut();
             let mut client_block_ref_ids: Vec<ClientID> =
                 self.blocks.clients.keys().cloned().collect();
-            client_block_ref_ids.sort_by(|a, b| b.cmp(a));
+            client_block_ref_ids.sort();
```

Comment:
Something I've missed - a microoptimisation used in Yjs: apply blocks from highest clients to lowest in order to avoid block skipping in case of potential conflict resolution in the same-place concurrent inserts.